ui/src/app/cohort-search/search-group/search-group.component.spec.ts

```diff
@@ -23,7 +23,7 @@ const itemA = fromJS({
   id: 'itemA',
   count: null,
   isRequesting: false,
-  type: 'icd9',
+  type: 'ICD9',
```

Comment:
Same for every type in this test as well.

Reply:
Done.